### PR TITLE
QA changes - Setting dates and Weekday saving validation

### PR DESCRIPTION
**Pull request by kumarp-v**

### CODE REVIEW

**Review by Manjot-Vienna — APPROVED** [review]
OK

**Review by Manjot-Vienna — CHANGES_REQUESTED** [review]
Need to do changes

File: ModelLibrary/Model/MPaymentTerm.cs [hunk 1]
```diff
@@ -1171,6 +1171,13 @@ protected override bool BeforeSave(bool newRecord)
                 }
             }
 
+            //** If schedule line are present system should not allow to save header with week day ** Dt: 02/04/2021 ** Modified By: Kumar ** //
+            if (Util.GetValueOfInt(GetWeekOffset()) > 0 && count > 0)
+            {
+                log.SaveError("Error", Msg.GetMsg(GetCtx(), "VIS_ScheduleExist"));
```
Comment (Manjot-Vienna) [inline ModelLibrary/Model/MPaymentTerm.cs]: log.SaveError("VIS_ScheduleExist", null);

Comment (kumarp-v) [reply ModelLibrary/Model/MPaymentTerm.cs]: As discussed, modified it as below.
log.SaveError("VIS_ScheduleExistForWeekDay", string.Empty);

**Review by Manjot-Vienna — APPROVED** [review]
Ok

**Pull request merged**